tables/TaQL/TableParse.cc

```diff
@@ -242,155 +252,162 @@ TableParseSelect::~TableParseSelect()
 
 
 //# Construct a TableParse object and add it to the container.
-void TableParseSelect::addTable (Int tabnr, const String& name,
-                                 const Table& ftab,
-                                 const String& shorthand,
-                                 Bool addToFromList,
-                                 const vector<const Table*> tempTables,
-                                 const vector<TableParseSelect*>& stack)
+Table TableParseSelect::addTable (Int tabnr, const String& name,
+                                  const Table& ftab,
+                                  const String& shorthand,
+                                  Bool addToFromList,
+                                  const vector<const Table*>& tempTables,
```

Comment:
Is this a `std::vector`? If so I prefer to name it `std::vector` instead of `vector`. Knowing it's a `std::vector` makes the code easier to understand.